### ISSUE TITLE
fluxcalib crash

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#1155]
using desispec/0.39.1
```
desi_compute_fluxcalibration --delta-color-cut 0.1 \
  --infile /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20201220/00069049/frame-b9-00069049.fits \
  --sky /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20201220/00069049/sky-b9-00069049.fits \
  --fiberflat /global/cfs/cdirs/desi/spectro/redux/cascades/calibnight/20201220/fiberflatnight-b9-20201220.fits \
  --models /global/cfs/cdirs/desi/spectro/redux/cascades/exposures/20201220/00069049/stdstars-9-00069049.fits \
  --outfile $SCRATCH/fluxcalib-b9-00069049.fits 
...
Traceback (most recent call last):
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/bin/desi_compute_fluxcalibration", line 4, in <module>
    __import__('pkg_resources').run_script('desispec==0.39.1', 'desi_compute_fluxcalibration')
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/conda/lib/python3.8/site-packages/pkg_resources/__init__.py", line 667, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/conda/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1464, in run_script
    exec(code, namespace, namespace)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/lib/python3.8/site-packages/desispec-0.39.1-py3.8.egg/EGG-INFO/scripts/desi_compute_fluxcalibration", line 16, in <module>
    fluxcal.main(args)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/lib/python3.8/site-packages/desispec-0.39.1-py3.8.egg/desispec/scripts/fluxcalibration.py", line 196, in main
    fluxcalib = compute_flux_calibration(frame, model_wave, model_flux, model_fibers%500, highest_throughput_nstars = args.highest_throughput)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/lib/python3.8/site-packages/desispec-0.39.1-py3.8.egg/desispec/fluxcalibration.py", line 1122, in compute_flux_calibration
    calibration[w]=cholesky_solve(A_pos_def, B[w])
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desispec/0.39.1/lib/python3.8/site-packages/desispec-0.39.1-py3.8.egg/desispec/linalg.py", line 28, in cholesky_solve
    X = scipy.linalg.cho_solve((UorL,lower),B)
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/conda/lib/python3.8/site-packages/scipy/linalg/decomp_cholesky.py", line 207, in cho_solve
    x, info = potrs(c, b1, lower=lower, overwrite_b=overwrite_b)
_flapack.error: (shape(c,0)==shape(b,0)) failed for 2nd argument b
```

**Comment by julienguy** [comment]
This is solved if running with version in PR #1157.

**Comment by sbailey** [comment]
Previously crashing test case now works, I think due to PR #1157 (Simplify fluxcalib).  Closing.
